code/firmware/rosco_m68k_v2.1/main1.c

```diff
@@ -71,10 +72,10 @@ noreturn void main1();
 
 noreturn void linit() {
   // copy .data
-  for (uint32_t *dst = &_data_start, *src = &_code_end; dst < &_data_end; *dst = *src, dst++, src++);
+  for (uint16_t *dst = &_data_start, *src = &_code_end; dst < &_data_end; *dst = *src, dst++, src++);
```

Comment:
Interesting. Why this change?

Reply:
Okay, yep, good points (and well spotted!). Given that I’d go with keeping this change 👍